test/src/mocha-utils.ts

```diff
@@ -332,3 +331,11 @@ export const createTimeout = <T>(
     }, n);
   });
 };
+
+export function rimrafSync(target: string): void {
```

Comment:
Let's use a different name, as `rimraf` is only associated with package. `recursiveDeleteSync` maybe?